markhor_tracks/src/markhor_hw_interface.cpp

```diff
@@ -248,6 +252,19 @@ void MarkhorHWInterface::write()
 
 void MarkhorHWInterface::read(const ros::Duration& period)
 {
+
+  //{ "flipper_fl_motor_j", "flipper_fr_motor_j", "flipper_rl_motor_j", "flipper_rr_motor_j" };
+
+  joint_position_[0] = -2*3.14159265358*front_left_drive->GetSensorCollection().GetPulseWidthPosition() / (double)track_encoder_to_rad_coeff;
```

Comment:
Pi probably exists somewhere as a constant or could be created.  Maybe 2*pi could be a constant with a representative name, it's not instantly clear what it is.
Be aware that every hard coded value ever will get this type of comment eventually.

Reply:
Good catch, I forgot it there, I'll use the pi defined in math.h